### PR TITLE
Story Promo - Fallback spacing for Leading promo

### LINKED ISSUE #3149: Story Promo - Leading variant needs fallback gutter
**Describe the bug**
When viewing in IE11 (that doesn't support CSS Grid), we do not have a gutter - 1rem of spacing between the image block and the text block

Screenshot 2020-02-17 at 12 54 27

**To Reproduce**
Steps to reproduce the behaviour:

1. Go to [Storybook example](https://bbc.github.io/psammead/iframe.html?id=components-storypromo-storypromo--leading-promo&knob-Select%20a%20service=persian&knob-Media%20Type=No%20media&knob-Image%20alt%20text=Robert%20Downey%20Junior%20in%20Iron%20Man&knob-Image%20src=https://ichef.bbci.co.uk/news/660/cpsprodpb/11897/production/_106613817_999_al_.jpg)
2. Adjust window so that text fills the whole text block
3. See text touching the image without necessary spacing. 

**Expected behaviour**
To have a 1rem space between the image and text block

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Testing notes**

- [ ] This bug fix is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by sareh**
Resolves #3149

**Overall change:** Leading story promo - ensures there's spacing between image and text blocks when viewing in browsers that don't support CSS Grid (screenshots below are in IE11)

**Before** [Link to live storybook example with Persian text](https://bbc.github.io/psammead/iframe.html?id=components-storypromo-storypromo--leading-promo&knob-Select%20a%20service=persian&knob-Media%20Type=No%20media&knob-Image%20alt%20text=Robert%20Downey%20Junior%20in%20Iron%20Man&knob-Image%20src=https://ichef.bbci.co.uk/news/660/cpsprodpb/11897/production/_106613817_999_al_.jpg)
Screenshot 2020-02-17 at 12 54 27

**After** [Link to local storybook example with Persian text](http://localhost:8180/iframe.html?id=components-storypromo-storypromo--leading-promo&knob-Select%20a%20service=persian&knob-Media%20Type=No%20media&knob-Image%20alt%20text=Robert%20Downey%20Junior%20in%20Iron%20Man&knob-Image%20src=https://ichef.bbci.co.uk/news/660/cpsprodpb/11897/production/_106613817_999_al_.jpg)

Screenshot 2020-02-19 at 16 33 30

**Code changes:**

- Add padding onto Text and Image blocks on the appropriate side to create a 'gutter' between the text and the image. e.g. on a RTL service like persian above, the Text block has a `padding-left: 0.5rem;` and the Image block has a `padding-right: 0.5rem;`
- Ensure `dir` prop is being passed to Image and Text blocks, so the correct padding can be added. 

- Remove `position: relative` from top-level StoryPromoWrapper - this allows the Image and Text blocks to remain within block flow and therefore respect width & padding

Following this change, I checked all three promo types - Top Regular and Leading, with a media video variant, to ensure layout is as expected for each of them. All appear as expected. 
Screenshot 2020-02-20 at 11 17 03

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated jest tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

File: packages/components/psammead-story-promo/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 | ------- | ----------- |
+| 4.0.0-alpha.13 | [PR#x](https://github.com/bbc/psammead/pull/x) Fix fallback width for Leading promo |
```
Comment (DenisHdz) [inline packages/components/psammead-story-promo/CHANGELOG.md]:
```suggestion
| 4.0.0-alpha.13 | [PR#3150](https://github.com/bbc/psammead/pull/3150) Fix fallback width for Leading promo |
```

**Review by DenisHdz — APPROVED** [review]
Checked with Browserstack and the new fallback values are applied correctly 👍 

leading-story-fallback

**Review by AlistairGempf — APPROVED** [review]
Looks good, based on the Chromatic snapshots and the code.

**Comment by PriyaKR** [comment]
@sareh i am seeing the text above the image on IE11 across all viewports verifies on browserstack.
Screen Shot 2020-02-20 at 11 43 45

**Comment by PriyaKR** [comment]
LGTM.

**Pull request merged**